js/screens/SignUp/SignUpScreen.js

```diff
@@ -48,38 +44,26 @@ class SignUpScreen extends Component {
   };
 
   addUser() {
-    const validEmail = /^([\w-.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/;
     const errorMessage = {
       code: 'Invalid email or password',
```

Comment:
This error message can be inside the helper function. If the email is not valid return this error.